### DISCUSSION TITLE
I added a promotional article about macchina and instructions on how to install it to x-cmd

### DISCUSSION THREAD

**Discussion by lunrenyi** [grtcdr/macchina#336] [Show and tell]
- Hi, [x-cmd](https://www.x-cmd.com/) is a **toolbox for Posix Shell**, offering a lightweight package manager built using shell and awk.. It helps you download macchina release packages from the internet and extract them into a unified directory for management, without requiring root permissions.

- **I mean, can the installation method provided by x-cmd be added to the macchina installation.md?**[The installation method for the x command](https://www.x-cmd.com/start/).
  ```sh
  x env use macchina
  ```

- We wrote a [macchina introduction article and a demo](https://www.x-cmd.com/pkg/macchina)

**Comment by grtcdr** [comment]
Hi, thanks!

To answer your question about adding x-cmd to the wiki: When we add an installation method to the wiki, we are essentially promoting it as an installation source, since I have never used or heard of x-cmd before this post, I don't feel comfortable adding it at the moment. That may change in the future when x-cmd is an established and trusted installation method.

**Reply by lunrenyi** [reply]
I understand. Thank you for your response.
